### ISSUE TITLE
Pylint output window is not inheriting the dark theme

### ISSUE THREAD

**Issue by oscargus** [spyder-ide/spyder#9644]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [ ] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [x] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [ ] Reset preferences with ``spyder --reset``
    * [ ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [ ] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
The Pylint output window is not dark using the dark theme (on my Windows 10 machine), when running the latest version of the master branch. Both with automatic from the default startup (which gives dark) and explicit dark theme (after restart).

![image](https://user-images.githubusercontent.com/8114497/59966228-7fa8ca00-9519-11e9-989e-284f7ad3d382.png)

Not a big issue, but since I noted that this was an issue mentioned in #8068 so it seems to have been fixed at some stage. 

### What steps reproduce the problem?

1. Clone repository
2. Follow contributing guidelines to install dependencies (with the exception of cloning my newly forked copy and optioning for Python 3.7 instead of 3.6)
3. Run `python bootstrap.py`

(Btw, `conda install -c spyder-ide spyder-kernels=1.*` did not install anything additional.)

### What is the expected output? What do you see instead?

A dark themed Pylint output window.

## Versions

* Spyder version: 4.0.0.dev0
* Python version: 3.7.3 64-bit
* Qt version: 5.9.6
* PyQt version: 5.9.2
* Operating System name/version: Windows 10

### Dependencies

```
pygments >=2.0     :  2.4.2 (OK)
qdarkstyle >=2.6.4 :  2.7 (OK)
sphinx >=0.6.6     :  2.1.0 (OK)
pyls >=0.19.0;<0.25:  0.24.0 (OK)
nbconvert >=4.0    :  5.5.0 (OK)
pandas >=0.13.1    :  None (NOK)
numpy >=1.7        :  None (NOK)
sympy >=0.7.3      :  None (NOK)
cython >=0.21      :  None (NOK)
qtconsole >=4.5.0  :  4.5.1 (OK)
IPython >=4.0      :  7.5.0 (OK)
matplotlib >=2.0.0 :  None (NOK)
pylint >=0.25      :  2.3.1 (OK)

```

**Comment by ccordoba12** [comment]
Thanks for reporting. We'll take a look at this in a future beta.

**Comment by ccordoba12** [comment]
@dalthviz, please take a look at this one.

**Comment by ccordoba12** [comment]
Fixed by PR #9877
